apps/authentication/api/connection_token.py

```diff
@@ -116,6 +146,8 @@ def get_rdp_file_info(self, token: ConnectionToken):
         rdp = token.asset.platform.protocols.filter(name='rdp').first()
         if rdp and rdp.setting.get('console'):
             rdp_options['administrative session:i'] = '1'
+        rdp_connection_speed = token.connect_options.get('rdp_connection_speed', 'auto')
+        rdp_options.update(RDP_CONNECTION_SPEED_OPTION_MAP.get(rdp_connection_speed, {}))
 
         # 文件名
         name = token.asset.name
```

Comment:
The main issue I found with this code snippet is that it contains several typos and missing or extra spaces which affect readability.

Here are some suggested changes:

1. Change the comment above Line 7 to English (it's currently German).
2. Replace `@@@` with an actual newline separator between different sections of the code.
3. Move all variable assignments to their own lines for improved clarity.
4. Correct indentation errors, especially at line 5, where it should be indented appropriately.
5. Add comments to explain what each section does; unnecessary blank lines can often clutter the documentation text block.

Some parts also look like they have typo in them as well:

- It should be "is_true" instead of just "bool".
- In "RDP_CONNECTION_SPEED_OPTIONS", there might not be a space after "R". Also, remove trailing spaces around `option`.

Let me know if you want these modifications applied or if you need anything else.